src/test/java/com/google/step/servlets/DealPostServletTest.java

```diff
@@ -104,7 +105,8 @@ public void testDoPost_success() throws IOException {
             eq(DATE_B),
             eq(SOURCE_A),
             anyLong(),
-            eq(RESTAURANT_ID_A)))
+            eq(RESTAURANT_ID_A),
+            anyList()))
```

Comment:
Assert empty list here (same below) until you start getting it from the request.